src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java

```diff
@@ -114,42 +109,124 @@ private String getBlobPath(String key, String path) {
     @Override
     public void archive(FilePath workspace, Launcher launcher, BuildListener listener, Map<String, String> artifacts)
             throws IOException, InterruptedException {
-        LOGGER.log(Level.FINE, "Archiving from {0}: {1}", new Object[] { workspace, artifacts });
-        Map<String, URL> artifactUrls = new HashMap<>();
+        LOGGER.log(Level.FINE, "Archiving from {0}: {1}", new Object[]{workspace, artifacts});
         BlobStore blobStore = getContext().getBlobStore();
+        if (artifacts.isEmpty()) {
```

Comment:
Under what conditions would this clause be hit? Normally the caller should skip and warn.

Reply:
I remove this condition.